content/en/docs/tasks/traffic-management/mirroring/index.md

```diff
@@ -137,10 +137,11 @@ In this step, you will change that behavior so that all traffic goes to `v1`.
 
 1.  Create a default route rule to route all traffic to `v1` of the service:
 
-    {{< warning >}}
-    If you installed/configured Istio with mutual TLS authentication enabled, you must add a TLS traffic policy `mode: ISTIO_MUTUAL` to the `DestinationRule` before applying it.
-    Otherwise requests will [generate 503 errors](/docs/ops/common-problems/network-issues/#503-errors-after-setting-destination-rule).
-    {{< /warning >}}
+    {{< tip >}}
```

Comment:
I think this tip will confuse readers, esp ones that don't understand the history. I suggest just removing the warning and saying nothing else.

Reply:
works for me, removed